ui/src/app/views/workspace/component.html

```diff
@@ -1,132 +1,171 @@
-        <clr-dg-action-overflow>
-          <button class="action-item"
-            [routerLink]="['cohorts', 'build']"
-            [queryParams]="{criteria: cohort.criteria}">
-            <clr-icon shape="copy"></clr-icon>
-            Clone
-          </button>
-          <button class="action-item"
-            routerLink="{{document.location.pathname}}/cohorts/{{cohort.id}}/edit">
-            <clr-icon shape="pencil"></clr-icon>
-            Edit
-          </button>
-          <button class="action-item"
-            routerLink="{{document.location.pathname}}/cohorts/{{cohort.id}}/review">
-            <clr-icon shape="grid-view"></clr-icon>
-            Review
-          </button>
-        </clr-dg-action-overflow>
-        <clr-dg-cell>{{cohort.name}}</clr-dg-cell>
-        <clr-dg-cell>{{cohort.description}}</clr-dg-cell>
-        <clr-dg-cell>{{cohort.creationTime | date:'medium'}}</clr-dg-cell>
-      </clr-dg-row>
-    </clr-datagrid>
-  </div>
-</div>
-<clr-modal [(clrModalOpen)]="cohortsError">
-  <h3 class="modal-title">Error:</h3>
-  <div class="modal-body">Could not fetch cohorts from workspace.</div>
-  <div class="modal-footer">
-    <button type="button" class="btn btn-primary" (click)="cohortsError = false">Ok</button>
-  </div>
-</clr-modal>
+<div class="workspace-container">
+  <div class="main-body">
+    <app-workspace-nav-bar></app-workspace-nav-bar>
+    <h2>{{workspace.name}}</h2>
+    <div style="display: flex; justify-content: space-between">
+      <div style="margin-left: -0.5rem;">
+        <button class="btn btn-link" [class.selected]="tabOpen === Tabs.Notebooks" (click)="tabOpen = Tabs.Notebooks">
+          Notebooks ({{notebookList.length}})
+        </button>
+        <button class="btn btn-link final" [class.selected]="tabOpen === Tabs.Cohorts" (click)="tabOpen = Tabs.Cohorts">
+          Cohorts ({{cohortList.length}})
+        </button>
+        <div class="alert {{alertCategory}}" *ngIf="showAlerts">
+          <div class="alert-items">
+            <div class="alert-item static">
+              <div class="alert-icon-wrapper">
+                <clr-icon class="alert-icon" shape="check-circle"></clr-icon>
+              </div>
+              <span class="alert-text">{{alertMsg}}</span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <button *ngIf="tabOpen===Tabs.Cohorts" class="btn btn-primary btn-create"
+                [class.awaiting-review]="awaitingReview" (click)="buildCohort()">
+          + Add a Cohort
+        </button>
+        <button *ngIf="tabOpen===Tabs.Notebooks" class="btn btn-primary btn-create"
+                [disabled]="clusterLoading || awaitingReview" (click)="newNotebook()">
+          + New Notebook
+        </button>
+      </div>
+    </div>
+    <!--<div class="area-block">-->
+    <!--<h3 class="cdr-text">CDR Version {{workspace.cdrVersionId}}</h3>-->
```

Comment:
You don't think we should show the CDR version, or you don't think we should show it in a block like this?

Reply:
Outside of the development team, no one is going to know what a "CDR" is. Also, we probably want the name rather than the DB primary key int, if anything.

Also I think this line is just dead code, since you include the CDR version below now.